lib/stormpath/rails/config/social_login_verification.rb

```diff
@@ -0,0 +1,41 @@
+module Stormpath
+  module Rails
+    module Config
+      class SocialLoginVerification
+        attr_reader :app,
+                    :facebook_app_id,
+                    :facebook_app_secret,
+                    :github_app_id,
+                    :github_app_secret,
+                    :google_app_id,
+                    :linkedin_app_id
+
+        def initialize(app_href)
+          @app = Stormpath::Rails::Client.client.applications.get(app_href)
+          initialize_directories
+        end
+
+        private
+
+        def initialize_directories
```

Comment:
Select didn't work for some reason. It didn't select all the account stores even if I did

```
app.account_store_mappings.select do |mapping|
  mapping.account_store if mapping.account_store.class == Stormpath::Resource::Directory
end.each do |directory|
```

and since the `search` cannot be applied, I thought I'd leave it as it is currently.

Try checking out why selecting didn't work. Really odd behaviour...